app/src/lib/git/diff.ts

```diff
@@ -474,6 +530,13 @@ export async function convertDiff(
     }
   }
 
+  if (diff.contents.includes(LFSVersionString)) {
+    // we must check for image file because someone can hold text files in LFS
+    if (imageFileExtensions.has(extension)) {
+      return getLFSImageDiff(repository, file, diff.hunks[0])
+    }
+  }
```

Comment:
```suggestion
   // we must check for image file because someone can hold text files in LFS
  if (imageFileExtensions.has(extension) && diff.contents.includes(LFSVersionString)) {
      return getLFSImageDiff(repository, file, diff.hunks[0])
  }
```

Nit: I don't think we have need for a nested `if` anymore. I also put the image file check first since likely less image files than non-image files in a typical repo and we only care about checking LFSVersionString on images.